Kitodo/src/main/java/org/kitodo/production/services/data/ProcessService.java

```diff
@@ -647,7 +648,7 @@ public List<ProcessDTO> findByAnything(String searchQuery) throws DataException
                 ProcessTypeField.PROJECT_TITLE.getKey(),
                 ProcessTypeField.COMMENTS.getKey(),
                 ProcessTypeField.WIKI_FIELD.getKey(),
-                ProcessTypeField.TEMPLATE_TITLE.getKey()).operator(Operator.AND);
+                ProcessTypeField.TEMPLATE_TITLE.getKey()).operator(Operator.AND).lenient(true);
```

Comment:
Just out of curiosity: why is this necessary?

Reply:
The id field of the process is stored as type "long" in the elasticsearch index "kitodo_process". When you query for a very long number in the ID field (which happens automatically when you search for all fields in the top left or search for parent processes), Elasticsearch tries to cast this number to a Long which does not work. Lenient true ignores exceptions regarding type casting.